README.md

```diff
@@ -78,8 +79,10 @@ Similar to Android's `signingConfigs` you need to setup so called `playAccountCo
 ```groovy
 android {
     playAccountConfigs {
-        serviceAccountEmail = 'your-service-account-email'
-        pk12File = file('key.p12')
+       defaultAccountConfig{
```

Comment:
Additionally, the indent seems odd. We prefer a 4-spaces indent for the code samples.